t/uv.t

```diff
@@ -48,6 +48,14 @@ bootstrap Devel::PPPort;
 
 package main;
 
+# skip tests on 5.006 and earlier
+if ( "$]" < 5.007 ) {
```

Comment:
This test passes with 5.6.2 and 5.6.1. So you need to skip it only for 5.6.0 and older.

```suggestion
if ( "$]" le '5.006' ) {
```

Reply:
ok for `5.6.0`, I'm not sure why we trend to prefer string comparison instead of integer, are we afraid of any bad behavior with UV?

Reply:
5.006001 is floating point number and it may lost precision when doing floating point comparison. In FP you cannot represent number 5.006001 precisely.